### DISCUSSION TITLE
Somewhat neuro-typical but couldn't pass the 'human' tests for setting up SMS 2FA

### DISCUSSION THREAD

**Discussion by winginitau** [community/community#106400] [Accessibility]
### Select Topic Area

Product Feedback

### Body

Fully appreciate that we're trying to keep AI out of github. Logged in with authenticator via base machine rather than virtual, presented to confirm details, decided to set up SMS 2FA. Sure, its late in the day. Could not work out the "count the objects puzzle" at all. After a few fails, tried 'audio' version, blew my mind. If we need that much stimulant to compete with AI, may as well give up now. IMHO, others clearly passed.

**Comment by ghostinhershell** [comment]
Hi @winginitau, 

Thanks so much for reaching out in the community. I know you initially shared this feedback last month. Were you able to complete the puzzles so that you could setup 2FA for your account?

**Comment by cehfisher** [comment]
Hello @winginitau - we hope you were able to resolve your 2FA issue. If not, we suggest opening a ticket on our [Support page](https://support.github.com/contact) for answers to account-specific questions and direct assistance. Please open a ticket and we will follow-up with next steps. Thanks!
